CMSgov/onemac
# Carlos oy220 test

Changes:
1. Changed Cognito to not allow signups.  This protects us from someone creating new accounts.
2. Create test accounts (see deploy.sh).
3. Updated UI to show or hide dev login button.

Test:
1. Verify the build is successful.
2. Login to AWS, view the Cognito user pool for this branch and verify that all test users are confirmed.
![image](https://user-images.githubusercontent.com/63804190/98281282-8f884900-1f6a-11eb-919a-4d5c6ffc08c9.png)

3. Access the SPA form and login with one of the test users:
user1@cms.hhs.local
 user2@cms.hhs.local
 user3@cms.hhs.local
 user4@cms.hhs.local
 user5@cms.hhs.local

Password: Passw0rd!
4. Verify you can login to the application.
5. Verify you can submit and view a change request.
6. Logout using the Dev Logout button.
7. Login using your work email (must be the same one used to commit to Git).  Password is Passw0rd!.  Verify you can login.

Endpoint: https://dklf1hjjrvkuy.cloudfront.net
